.github/workflows/backend.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         with:
           m2-cache-key: 'kondo'
       - name: Run clj-kondo
-        run: ./bin/kondo.sh
+        run: ./bin/mage kondo
```

Comment:
Ran and passed, see: [Run tests / backend-tests / be-tests-result (pull_request)](https://github.com/metabase/metabase/actions/runs/14652434376/job/41124007178?pr=57229)Successful in 3s